src/miqclient/api.py

```diff
@@ -21,7 +21,7 @@ class APIException(Exception):
 
 
 class API(object):
-    def __init__(self, entry_point, auth, logger=None):
+    def __init__(self, entry_point, auth, logger=None, verify=True, certfile_path=None):
```

Comment:
Consider s/verify/verify_tls/.
And maybe s/certfile_path/ca_bundle_path/?  Not sure if that's the best name, but a particular drawback of calling it "certfile" is that `requests` supports both file and directory bundles.

It's worth documenting that if the bundle path is specified it *replaces* the system trust roots.  If you just want to add some cert, you have to concat it to the system's CA bundle.